nucliadb_utils/nucliadb_utils/settings.py

```diff
@@ -49,8 +50,25 @@ class HTTPSettings(BaseSettings):
 http_settings = HTTPSettings()
 
 
+class FileBackendConfig(str, Enum):
+    gcs = "gcs"
```

Comment:
We usually use upper case for enum variants (variable name). Should we keep this convention?

Reply:
Ah yes! Good point. I'll update